src/po/Make_mvc.mak

```diff
@@ -53,43 +517,73 @@ PO_INPUTLIST = \
 	gvim.desktop.in \
 	vim.desktop.in
 
-files: $(PO_INPUTLIST)
+files : $(PO_INPUTLIST)
 	$(LS) $(LSFLAGS) $(PO_INPUTLIST) > .\files
 
-first_time: files
-	$(VIM) -u NONE --not-a-term -S tojavascript.vim $(LANGUAGE).pot $(PO_VIM_INPUTLIST)
+first_time : files
+	$(VIMEXE) -u NONE --not-a-term -S tojavascript.vim $(LANGUAGE).po \
+		$(PO_VIM_INPUTLIST)
 	set OLD_PO_FILE_INPUT=yes
 	set OLD_PO_FILE_OUTPUT=yes
-	$(XGETTEXT) --default-domain=$(LANGUAGE) --add-comments $(XGETTEXT_KEYWORDS) --files-from=.\files $(PO_VIM_JSLIST)
-	$(VIM) -u NONE --not-a-term -S fixfilenames.vim $(LANGUAGE).pot $(PO_VIM_INPUTLIST)
+	$(XGETTEXT) --default-domain=$(LANGUAGE) --add-comments $(XGETTEXT_KEYWORDS) \
+		--files-from=.\files $(PO_VIM_JSLIST)
+	$(VIMEXE) -u NONE --not-a-term -S fixfilenames.vim $(LANGUAGE).po \
+		$(PO_VIM_INPUTLIST)
 	$(RM) *.js
 
-$(PACKAGE).pot: files
-	$(VIM) -u NONE --not-a-term -S tojavascript.vim $(PACKAGE).pot $(PO_VIM_INPUTLIST)
+$(PACKAGE).pot : files
+	$(VIMEXE) -u NONE --not-a-term -S tojavascript.vim $(PACKAGE).pot \
+		$(PO_VIM_INPUTLIST)
 	set OLD_PO_FILE_INPUT=yes
 	set OLD_PO_FILE_OUTPUT=yes
-	$(XGETTEXT) --default-domain=$(PACKAGE) --add-comments $(XGETTEXT_KEYWORDS) --files-from=.\files $(PO_VIM_JSLIST)
+	$(XGETTEXT) --default-domain=$(PACKAGE) --add-comments $(XGETTEXT_KEYWORDS) \
+		--files-from=.\files $(PO_VIM_JSLIST)
 	$(MV) $(PACKAGE).po $(PACKAGE).pot
-	$(VIM) -u NONE --not-a-term -S fixfilenames.vim $(PACKAGE).pot $(PO_VIM_INPUTLIST)
+	$(VIMEXE) -u NONE --not-a-term -S fixfilenames.vim $(PACKAGE).pot \
+		$(PO_VIM_INPUTLIST)
 	$(RM) *.js
 
+# 06.11.23, added by Restorer
+# When updating ja.sjis.po there are a bunch of errors and a crash.
+# The files that are converted to a different encoding clearly state "DO NOT EDIT".
+update-po : $(MOFILES:.mo=)
+
 # Don't add a dependency here, we only want to update the .po files manually
-$(LANGUAGES):
+$(LANGUAGES) :
```

Comment:
yes agree

Reply:
> If the name of the last target before the colon (:) is a single character, you must
put a space between the name and the colon; otherwise, NMAKE interprets the
letter-colon combination as a drive specifier.

Ah, I didn't know this pitfall. (I think this is a very rare case, though.)

Reply:
Yes, I agree with you that the case is quite rare. And we are unlikely to encounter this situation.
But the Russians have a saying «иногда лучше перебдеть, чем недобдеть».
Which means that it is better to try to foresee all possible situations now than to deal with unpleasant consequences later.
Maybe someone wants to make a translation for C.

Reply:
I'd prefer if we stay consistent here with other Makefiles and also avoid changing too many lines for no benefit

Reply:
unimportant for now. withdrawn whitespace